ProcessMessage/index.ts

```diff
@@ -79,6 +80,7 @@ const activityFunctionHandler: AzureFunction = getProcessMessageHandler({
   lProfileModel: profileModel,
   lServicePreferencesModel: servicePreferencesModel,
   optOutEmailSwitchDate: config.OPT_OUT_EMAIL_SWITCH_DATE,
+  pendingActivationGracePeriod: config.PENDING_ACTIVATION_GRACE_PERIOD_SECONDS as Second,
```

Comment:
not sure we need this explicit cast

Reply:
Good catch, the cast is needed but is wrong, because the casting is not enough. To convert a number to a `Second` type we need even the `kind` prop.
This line should be changed in
```suggestion
  pendingActivationGracePeriod: Object.assign(
          DEFAULT_PENDING_ACTIVATION_GRACE_PERIOD_SECONDS,
          { kind: "Second" as const }
        ),
```
What do you think about?

Reply:
It should work but a test could help us in this case :D

Reply:
why only DEFAULT_PENDING_ACTIVATION_GRACE_PERIOD_SECONDS ?

Reply:
Sorry wrong code paste, replace `DEFAULT_PENDING_ACTIVATION_GRACE_PERIOD_SECONDS` with `DEFAULT_PENDING_ACTIVATION_GRACE_PERIOD_SECONDS`. The unit test already present works correctly passing the Second object.

Reply:
Fixed in 4f8aee6

Reply:
Not sure about it, see [this post](https://medium.com/@KevinBGreene/surviving-the-typescript-ecosystem-branding-and-type-tagging-6cf6e516523d) about Branded Types: they only works at compile time and defined using _"as"_

Reply:
Good point, but i think that `Second` is not a _Branded Type_ [link](https://github.com/pagopa/io-ts-commons/blob/master/src/units.ts)

Reply:
I think it is exactly a Branded Type, even if we called it `kind` and not `__brand`

Reply:
Restored casting in 77022a0